dev/devicelab/lib/tasks/plugin_tests.dart

```diff
@@ -84,6 +84,12 @@ class PluginTest {
           section('Test app');
           await app.runFlutterTest();
         }
+        // Validate local engine handling. Currently only implemented for macOS.
```

Comment:
Alternately I could make this a new top-level `PluginTest` parameter and run it separately in the various plugin test tasks, but it's so fast to run this seemed easier.